src/Dispatcher.cpp

```diff
@@ -145,6 +145,16 @@ void Dispatcher::OnUpgradeCompleted(sc2::UpgradeID id_) {
         i->OnUpgradeCompleted(id_);
 }
 
+void Dispatcher::OnUnitEnterVision(const sc2::Unit* unit_) {
+//    gHistory.info() << sc2::UnitTypeToName(unit_->unit_type) <<
```

Comment:
Why this part was commented out?

Reply:
It's triggered too often during big fights and slows the game. It's not just first time seen, but every time. If thought to leave it in commented out, so authors can easily uncomment it of they need to use for debugging, but maybe that isn't needed.

Reply:
I think better to remove it, but ok.